CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
+## 0.2.13 - 2017/01/25
+
+* [BUGFIX] Correctly fetch data for videos that belong to a Facebook page with a username that contains URL-unsafe characters. For instance "https://www.facebook.com/KinoToPrzygoda /" (with a space) is a valid Facebook page URL.
+
+
 ## 2017/01/24
```

Comment:
@bruce-fs This PR is now perfect. But can you remove this date here. All these changes will fall under `0.2.13`.